src/components/patch-details/patch-details.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { Link as RouterLink } from "react-router-dom";
+
+import Cookies from "js-cookie";
+
+import { locale } from "../../services";
+
+import { Asset } from "@mtfh/common/lib/api/asset/v1";
+import { Patch, usePatchOrArea } from "@mtfh/common/lib/api/patch/v1";
+import {
+  Button,
+  Heading,
+  SummaryList,
+  SummaryListItem,
+  Text,
+} from "@mtfh/common/lib/components";
+
+const PatchTable = ({ patches }: { patches: Patch[] }) => {
+  const assetPatch = patches.find((patch) => patch.patchType === "patch");
+  const parentId = assetPatch?.parentId || "";
+  const parentAreaReq = usePatchOrArea(parentId);
+  const parentArea = parentAreaReq.data;
```

Comment:
How does this work? Arent these hooks asynchronous?

Reply:
It is asynchronous, but as soon as the request resolves the area manager gets set on the page.
So it's "N/A" for a split second until the request comes back.
usePatchOrArea does not return a promise though so doesn't need to be used with .then() etc

Reply:
It is asynchronous, but as soon as the request resolves the area manager gets set on the page.
So it's "N/A" for a split second until the request comes back.
usePatchOrArea does not return a promise though so doesn't need to be used with .then() etc